NEWS.md

```diff
@@ -44,6 +44,8 @@
 
 14. `fread` loads `.bgz` files directly, [#5461](https://github.com/Rdatatable/data.table/issues/5461). Thanks to @TMRHarrison for the request with proposed fix, and Benjamin Schwendinger for the PR.
 
+15. `fwrite` now returns the file path after writing data, [#5706](https://github.com/Rdatatable/data.table/issues/5706). This enhancement allows users to easily capture and utilize the file path for subsequent operations.Thanks to @Nj221102 for the PR.
```

Comment:
space between 'operations.' and 'Thanks'

Reply:
Also, @eliocamp should be credited

Reply:
sure :)